services/server/src/server/server.ts

```diff
@@ -62,10 +80,35 @@ export class Server {
     verificationServiceOptions: VerificationServiceOptions,
     storageServiceOptions: StorageServiceOptions,
   ) {
+    if (options.logLevel) {
+      if (options.logLevel && !validLogLevels.includes(options.logLevel)) {
```

Comment:
```suggestion
    if (options.logLevel !== undefined) {
      if (!validLogLevels.includes(options.logLevel)) {
```

the logLevel can be 0